### PR TITLE
[JENKINS-58486] Add help files to the plugin

### PR DESCRIPTION
**Pull request by AbhyudayaSharma**
Adds help files to the plugin and improves the 'Submit' buttons on the Web UI

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <version>${revision}${changelist}</version>
     <name>Folder-based Authorization Strategy</name>
     <description>Manage access to Folders</description>
+    <url>https://github.com/jenkinsci/folder-auth-plugin</url>
```
Comment (AbhyudayaSharma) [inline pom.xml]: Role Strategy plugin uses the Wiki page for the URL. Is it fine to use the link to the GitHub repo here?

File: src/main/webapp/css/folder-strategy.css [hunk 2]
```diff
@@ -62,19 +62,35 @@ div.role {
...
-    margin-top: 10px;
+    margin-top: 5px;
     margin-left: 10px;
-    background-color: #dddddd;
-    display: block;
+    line-height: 1.25;
+    border-style: solid;
+    border-color: #007bff;
+    background-color: #007bff;
+    color: #ffffff;
```
Comment (AbhyudayaSharma) [inline src/main/webapp/css/folder-strategy.css]: The buttons look like this now:

image

File: src/main/resources/index.jelly [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<div>
+    <p>
+        Easily configure permissions user permissions for
+        <a href="https://wiki.jenkins.io/display/JENKINS/CloudBees+Folders+Plugin">Folders</a>
```
Comment (oleg-nenashev) [inline src/main/resources/index.jelly]: Reference plugins.jenkins.io instead?

File: pom.xml (hunk 1 above)
Comment (oleg-nenashev) [reply pom.xml]: +1. We consider changing the documentation publishing logic to take this URL into account leter

File: src/main/resources/index.jelly [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<div>
+    <p>
+        Easily configure permissions user permissions for
```
Comment (oleg-nenashev) [inline src/main/resources/index.jelly]: Too long

File: src/main/resources/io/jenkins/plugins/folderauth/FolderBasedAuthorizationStrategy/help.html [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<p>
+    Allows configuring users' permissions for jobs organized in folders and agents connected to Jenkins.
+    Also provides support for permissions that are applicable everywhere inside Jenkins. Please visit the
+    <a href="https://github.com/jenkinsci/folder-auth-plugin" target="_blank">
```
Comment (AbhyudayaSharma) [inline src/main/resources/io/jenkins/plugins/folderauth/FolderBasedAuthorizationStrategy/help.html]: Is this better?

File: src/main/resources/index.jelly [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?jelly escape-by-default='true'?>
+<div>
+    <p>
+        Allows configuring users' permissions to projects organized in
+        folders from the Cloudbees Folder plugin. See the
+        <a href="https://github.com/jenkinsci/folder-auth-plugin">
+            GitHub page
```
Comment (AbhyudayaSharma) [inline src/main/resources/index.jelly]: Is this better?

Comment (oleg-nenashev) [reply src/main/resources/index.jelly]: Yes. GitHub link is probably not needed as well, Jenkins Update Center injects Wiki or GitHub link by default IIRC

Comment (AbhyudayaSharma) [reply src/main/resources/index.jelly]: I verified it, the title of the plugin gets the link to the GitHub page. Thanks!

**Pull request merged**